### PR TITLE
debugger: Shortend SPU/PPU thread names

### PR DESCRIPTION
**Pull request by elad335**
Before:
image

After:
image

This only affects debugger and does not affect the log.

### CODE REVIEW

File: rpcs3/Emu/Cell/lv2/sys_spu.cpp [hunk 1]
```diff
@@ -405,11 +405,11 @@ error_code sys_spu_thread_initialize(ppu_thread& ppu, vm::ptr<u32> thread, u32 g
 
 	verify(HERE), idm::import<named_thread<spu_thread>>([&]()
 	{
-		std::string full_name = fmt::format("SPU[0x%07x] Thread", tid);
+		std::string full_name = fmt::format("SPU[0x%07x] ", tid);
 
 		if (!thread_name.empty())
 		{
-			fmt::append(full_name, " (%s)", thread_name);
+			fmt::append(full_name, "%s ", thread_name);
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/lv2/sys_spu.cpp:412]:
```suggestion
			fmt::append(full_name, " %s", thread_name);
```

File: rpcs3/Emu/Cell/lv2/sys_spu.cpp [hunk 2]
```diff
@@ -405,11 +405,11 @@ error_code sys_spu_thread_initialize(ppu_thread& ppu, vm::ptr<u32> thread, u32 g
 
 	verify(HERE), idm::import<named_thread<spu_thread>>([&]()
 	{
-		std::string full_name = fmt::format("SPU[0x%07x] Thread", tid);
+		std::string full_name = fmt::format("SPU[0x%07x] ", tid);
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/lv2/sys_spu.cpp:408]:
```suggestion
		std::string full_name = fmt::format("SPU[0x%07x]", tid);
```

Comment (Megamouse) [reply rpcs3/Emu/Cell/lv2/sys_spu.cpp:408]: I think I'm missing something here

File: rpcs3/Emu/Cell/PPUModule.cpp [hunk 3]
```diff
@@ -1518,7 +1518,7 @@ void ppu_load_exec(const ppu_exec_object& elf)
 	p.stack_addr = vm::cast(vm::alloc(primary_stacksize, vm::stack, 4096));
 	p.stack_size = primary_stacksize;
 
-	auto ppu = idm::make_ptr<named_thread<ppu_thread>>("PPU[0x1000000] Thread (main_thread)", p, "main_thread", primary_prio, 1);
+	auto ppu = idm::make_ptr<named_thread<ppu_thread>>("PPU[0x1000000] main_thread ", p, "main_thread", primary_prio, 1);
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/PPUModule.cpp:1521]: Was the space intentional?

```suggestion
	auto ppu = idm::make_ptr<named_thread<ppu_thread>>("PPU[0x1000000] main_thread", p, "main_thread", primary_prio, 1);
```

File: rpcs3/Emu/Cell/lv2/sys_spu.cpp (hunk 1 above)
Comment (elad335) [reply rpcs3/Emu/Cell/lv2/sys_spu.cpp:412]: I actually want that space infront as I found it odd looking if the last character is a normal character in the rectangle with no blank space inbetween.
It also applies for threads with "empty" names.
I wonder if there's a qt way of doing it instead.

Comment (Megamouse) [reply rpcs3/Emu/Cell/lv2/sys_spu.cpp:412]: do you have a screenshot of the "weird" names

**Pull request merged**